docs/source/about/guides/linux/headless_ssh.rst

```diff
@@ -250,125 +249,102 @@ As an alternative to a dummy dongle, you can use this config to create a virtual
-#. Execute the following to update your sudoer config file.
+#. Reboot or run the following:
 
    .. code-block:: bash
 
-      echo "${USER} ALL=(ALL:ALL) ALL, NOPASSWD: $(pwd)/sunshine-setup.sh" \
-        | sudo tee /etc/sudoers.d/${USER}
-
-These changes allow the script to use sudo without being prompted with a password.
-
-e.g. ``sudo $(pwd)/sunshine-setup.sh``
+      sudo udevadm control --reload-rules && sudo udevadm trigger
 
+.. hint::
+   If you need to roll back your uinput udev rules, follow the linux install instructions again in :ref:`setup <about/setup:install>`
 
 Stream Launcher Script
 ^^^^^^^^^^^^^^^^^^^^^^
 
-This is the main entrypoint script that will run the ``sunshine-setup.sh`` script, start up X server, and Sunshine.
-The client will call this script that runs on the host via ssh.
+The client will execute this script on the host to startup Xorg and Sunshine.
 
 
 **Sunshine Startup Script**
 
 This guide will refer to this script as ``~/scripts/sunshine.sh``.
-The setup script will be referred as ``~/scripts/sunshine-setup.sh``
 
 .. code-block:: bash
 
-    #!/bin/bash
+   #!/bin/bash
 
-    export DISPLAY=:0
+   export DISPLAY=:0
 
-    # Check existing X server
-    ps -e | grep X >/dev/null
-    [[ ${?} -ne 0 ]] && {
+   dpi=${1:-144}
+   
+   # Check existing X server
+   ps -e | grep X >/dev/null
+   [[ ${?} -ne 0 ]] && {
+     echo "DPI: ${dpi}"
+     echo "Xft.dpi: ${dpi}" > $HOME/.Xresources
      echo "Starting X server"
      startx &>/dev/null &
      [[ ${?} -eq 0 ]] && {
        echo "X server started successfully"
      } || echo "X server failed to start"
-    } || echo "X server already running"
+   } || echo "X server already running"
+
+   # Give some time for X server to startup
+   sleep 3
 
-    # Check if sunshine is already running
-    ps -e | grep -e .*sunshine$ >/dev/null
-    [[ ${?} -ne 0 ]] && {
+   # Check if sunshine is already running
+   ps -e | grep -e .*sunshine$ >/dev/null
+   [[ ${?} -ne 0 ]] && {
      sudo ~/scripts/sunshine-setup.sh
      echo "Starting Sunshine!"
      sunshine > /dev/null &
      [[ ${?} -eq 0 ]] && {
        echo "Sunshine started successfully"
      } || echo "Sunshine failed to start"
-    } || echo "Sunshine is already running"
+   } || echo "Sunshine is already running"
+
+   # Add any other Programs that you want to startup automatically
+   # e.g.
+   # steam &> /dev/null &
+   # firefox &> /dev/null &
+   # kdeconnect-app &> /dev/null &
+
+Notice how we pass the "dpi" as a way to get a good scale level for the font and windows.
+You can change the default if "144" doesn't suit  your needs.
+The .xinitrc will use the .Xresources file to set the dpi.
 
-    # Add any other Programs that you want to startup automatically
-    # e.g.
-    # steam &> /dev/null &
-    # firefox &> /dev/null &
-    # kdeconnect-app &> /dev/null &
+This is what my .xinitrc file looks like:
+
+.. code-block:: bash
+
+   xrdb -merge ~/.Xresources
+   ...
```

Comment:
Are the `...` literal? If not someone will take them literally.

Reply:
lol